pint/unit.py

```diff
@@ -1109,6 +1110,9 @@ def _convert(self, value, src, dst, inplace=False):
         if isinstance(value, Decimal):
             factor = Decimal(str(factor))
 
+        if isinstance(value, Fraction):
+            factor = Fraction(str(factor))
```

Comment:
Yes but anyway if for instance the factor is 1.1 I want Fraction(factor) to evaluated as 11/10, not 2476979795053773/2251799813685248 (which happens due to the fact that 1.1 has no exact float representation?)

```python
Python 2.7.9 (default, Mar  1 2015, 12:57:24) 
[GCC 4.9.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from fractions import Fraction
>>> Fraction(1.1)
Fraction(2476979795053773, 2251799813685248)
>>> Fraction('1.1')
Fraction(11, 10)
```